torch/onnx/symbolic_caffe2.py

```diff
@@ -194,7 +194,7 @@ def upsample_nearest2d(
 @symbolic_helper.parse_args("v", "is", "is", "is", "is", "i")
 def max_pool2d(g, input, kernel_size, stride, padding, dilation, ceil_mode):
     if input not in symbolic_helper._quantized_ops:
-        return opset9.max_pool2d(
+        return opset9.max_pool2d(  # type: ignore[attr-defined]
```

Comment:
Curious, why is this needed?

Reply:
I see, these are no longer functions in `symbolic_opset9.py`. I guess this fall should fail?

Reply:
A: https://github.com/pytorch/pytorch/pull/84448#discussion_r966486511